### ISSUE TITLE
Can't set move_base params to allow paths to be planned into unknown space.

### ISSUE THREAD

**Issue by scploeger** [RobotnikAutomation/rbkairos_sim#17]
I am trying to add [explore-lite](http://wiki.ros.org/explore_lite) to this workspace to map an unknown area. The docs of explore-lite mention the requirement to "enable unknown space tracking by setting ```track_unknown_space: true```"

I have tried adding ```track_unknown_space: true``` under ```obstacle_laser_layer:``` as well as ```obstacle_camera_layer:``` within ```costmap_commom_params.yaml```.  I have also added ```allow_unknown: true``` to ```move_base_params.yaml```.

With this, I launch ```rbkairos_sim_bringup``` and then from within rviz try to set a 2D Nav Goal Outside the mapped area, but the terminal spits out a ```Failed to get a plan.``` error.

**Comment by alex-arnal** [comment]
From what I have seen, by default the configuration of move_base and costmaps does not allow navigation through non-explored spaces. I have tried setting the parameters that you mentioned but I couldn't make it work.  Disabling the global_costmap I have been capable to navigate through unknown spaces. I did the following:

1. I launched the simulation just with the navigation enabled:
```bash
roslaunch rbkairos_sim_bringup rbkairos_complete.launch localization_robot_a:=false amcl_and_mapserver_a:=false move_base_robot_a:=true
```

2. I launched the mapping process:
```bash
ROS_NAMESPACE=robot roslaunch summit_xl_localization slam_gmapping.launch
```
3. I dynamically reconfigured the enabled (to false) parameter for global_costmap:
```bash
rosrun rqt_reconfigure rqt_reconfigure
```
![image](https://user-images.githubusercontent.com/30115565/169802142-08414ae8-494c-48a7-9856-b615b0a940a6.png)
4. I sent goals using RVIZ:
![image](https://user-images.githubusercontent.com/30115565/169802667-e116b4c9-c933-4b9f-8260-eb80515aef5c.png)


I hope this solution works for your use case.
